iroha-transfer-asset/step1.md

```diff
@@ -1,10 +1,23 @@
-The sandbox runs Iroha v1.0 pre-configured with two users: `alice@test` and `bob@test`. Both having [all available permissions](https://github.com/hyperledger/iroha/blob/master/shared_model/schema/primitive.proto#L29) in the system. There is also an asset -- `coin#test`.
+This Iroha is pre-configured with two users: `alice@test` and `bob@test`. 
 
-There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system.
+![Alice and Bob](assets/alicebob.png)
+
+They both have [all available permissions](https://github.com/hyperledger/iroha/blob/master/shared_model/schema/primitive.proto#L29) in the system. 
+So both of them can perform any of the available actions in the system. 
+
+There is also an asset -- `coin#test`:
+
+![Coin](assets/coin.png)
+
+Alice, Bob and the asset are parts of the same domain - `test`. 
+There might be other ones in the system, representing different households, companies or institutions.
+
+There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so Iroha could process commands written within .py file. 
```

Comment:
Iroha itself does not process commands written in Python. We interact with Iroha using Python library.